test/unit/org/apache/cassandra/service/accord/SimulatedAccordCommandStoreTestBase.java

```diff
@@ -380,4 +396,81 @@ protected static Gen<Pair<Txn, FullRoute<?>>> randomTxn(Gen<Routable.Domain> dom
             }
         };
     }
+
+    public static class DepsModel
+    {
+        private final Map<RoutingKey, List<TxnId>> keyConflicts = new HashMap<>();
+        private final RangeTree<RoutingKey, Range, TxnId> rangeConflicts = RTree.create(RangeTreeRangeAccessor.instance);
+        private final Ranges storeRanges;
+
+        public DepsModel(Ranges storeRanges)
+        {
+            this.storeRanges = storeRanges;
+        }
+
+        public Map<RoutingKey, List<TxnId>> keyConflicts(Seekables<?, ?> keysOrRanges)
+        {
+            keysOrRanges = keysOrRanges.slice(storeRanges, Routables.Slice.Minimal);
```

Comment:
the store is the full range, but it has 2 ranges `(-Inf, ?], (?, +Inf]`, so this slice just makes sure to include the ranges we will see rather than requested